scripts/gnosis/VerifyProxyEtherscan.s.sol

```diff
@@ -49,9 +49,9 @@ contract VerifyProxyEtherscan is Utils {
     // @dev Deploys diamond and connects facets
     function _deployDiamondEtherscan() internal {
         // Deploy every facet
-        facetNames.push("DiamondEtherscanFacet");
-        // etherscanFacet = new DiamondEtherscanFacet();
-        etherscanFacet = DiamondEtherscanFacet(0xC492fBAe68cE6C5E14C7ed5cd8a59babD5c90e4C);
+        facetNames.push("DiamondEtherscan");
+        // etherscanFacet = new DiamondEtherscan();
```

Comment:
if you switch comments above we should also switch this one

Reply:
switched